RELEASING/release-notes-1-4/README.md

```diff
@@ -19,7 +19,7 @@ under the License.
 
 # Release Notes for Superset 1.4
 
-Superset 1.4 focuses heavily on continuing to polish the core Superset experience. This release has a very very long list of fixes from across the community.
+Superset 1.4 focuses heavily on continuing to polish the core Superset experience. This release has a very long list of fixes from across the community.
```

Comment:
I'm happy to drop this, but I don't think it added any real value. (I left one really long duplicate item.)